src/data/navigation/sections/guide.js

```diff
@@ -3,4 +3,130 @@ module.exports = [
+              title: "@depends",
+              path: "/guide/integration/annotations/magento-depends/",
+            },
+            {
+              title: "@magentoAppArea",
+              path: "/guide/integration/annotations/magento-app-area/",
+            },
+            {
+              title: "@magentoAppIsolation",
+              path: "/guide/integration/annotations/magento-app-isolation/",
+            },
+            {
+              title: "@magentoCache",
+              path: "/guide/integration/annotations/magento-cache/",
+            },
+            {
+              title: "@magentoComponentsDir",
+              path: "/guide/integration/annotations/magento-components-dir/",
+            },
+            {
+              title: "@magentoConfigFixture",
+              path: "/guide/integration/annotations/magento-config-fixture/",
+            },
+            {
+              title: "@magentoDataFixture",
+              path: "/guide/integration/annotations/magento-data-fixture/",
+            },
+            {
+              title: "@magentoDbIsolation",
+              path: "/guide/integration/annotations/magento-db-isolation/",
+            },
+          ],
+        },
+        {
+          title: "PHP built-in attributes",
+          path: "/guide/integration/attributes/",
+          pages: [
+            {
+              title: "AppArea",
+              path: "/guide/integration/attributes/app-area/",
+            },
+            {
+              title: "AppIsolation",
+              path: "/guide/integration/attributes/app-isolation/",
+            },
+            {
+              title: "Cache",
+              path: "/guide/integration/attributes/cache/",
+            },
+            {
+              title: "ComponentsDir",
+              path: "/guide/integration/attributes/components-dir/",
+            },
+            {
+              title: "ConfigFixture",
+              path: "/guide/integration/attributes/config-fixture/",
+            },
+            {
+              title: "DataFixture",
+              path: "/guide/integration/attributes/data-fixture/",
+            },
+            {
+              title: "DataFixtureBeforeTransaction",
+              path: "/guide/integration/attributes/data-fixture-before-transaction/",
+            },
+            {
+              title: "DbIsolation",
+              path: "/guide/integration/attributes/db-isolation/",
+            },
+            {
+              title: "IndexerDimensionMode",
+              path: "/guide/integration/attributes/indexer-dimension-mode/",
+            },
+          ],
+        },
+      ],
+    },
+    {
+      title: "JavaScript testing",
+      path: "/guide/js/",
+    },
+    {
+      title: "PHP unit testing",
+      path: "/guide/unit/",
+      pages: [
+        {
+          title: "Command line",
+          path: "/guide/unit/command-line/",
+        },
+        {
+          title: "PHPStorm",
+          path: "/guide/unit/phpstorm/",
```

Comment:
In the stage environment, I weirdly cannot click on this item in the left-nav, but it works fine locally, so probably just a deploy issue.

Reply:
How weird... works for me on staging 🤷